module/metrics.go

```diff
@@ -130,23 +131,34 @@ type HotstuffMetrics interface {
 	// HotStuffWaitDuration reports Metrics C6 HotStuff Idle Duration
```

Comment:
```suggestion
	// HotStuffWaitDuration reports Metrics C6 HotStuff Idle Duration - the time between receiving and 
	// enqueueing a message to beginning to process that message.
```
Updating doc based on what I think this metric should measure. As mentioned in previous comments, we currently don't measure this correctly.